client/components/profilePage/myLibrary.jsx

```diff
@@ -43,46 +43,66 @@ export default function myLibrary() {
 
   let searchBooks = function (e) {
     e.preventDefault();
-    axios.get(`https://www.googleapis.com/books/v1/volumes?q=inauthor:${searchInput}&maxResults=25`)
-    .then((results)=> {
-      let searchResults = results.data.items.map((book) => {
-        let bookInfo = {};
-        bookInfo.title= book.volumeInfo.title,
-        bookInfo.authors= book.volumeInfo.authors,
-        bookInfo.isbn= book.volumeInfo.industryIdentifiers[0].identifier;
-        bookInfo.description = book.volumeInfo.description;
-        bookInfo.image = book.volumeInfo.imageLinks ? book.volumeInfo.imageLinks.thumbnail : 'https://i.imgur.com/sJ3CT4V.gif';
-        bookInfo.price = book.saleInfo.listPrice ? `$${book.saleInfo.listPrice.amount}` : 'Not Available'
-        return bookInfo;
+    axios
+      .get(
+        `https://www.googleapis.com/books/v1/volumes?q=inauthor:${searchInput}&maxResults=25`
+      )
+      .then((results) => {
+        let searchResults = results.data.items.map((book) => {
+          let bookInfo = {};
+          (bookInfo.title = book.volumeInfo.title),
```

Comment:
did you mean to use commas here?

Reply:
Honestly not sure what is going on there. I think prettier might have reformatted this so it is showing up as a change (there are commas on the previous code to the left). I'll change it and push.